pkg/app/src/components/Account/modules/overviewTab.tsx

```diff
@@ -18,6 +17,7 @@ interface OverviewTabProps {
 
 const OverviewTab: FC<OverviewTabProps> = ({ accountState }) => {
```

Comment:
1. As always. Do not write FC components.
2. Do not pass AccountState or Values you get from a hook as parameter (avoid useless state management)
3. Use const address = useCurrentAccountAddress() to get the current account address.
4. Do not pass the balance state to MyBalancesCard. -> Write the hook in the component if you don't need the data in the overview table(parent). (avoid useless state management)